hedera-node/hedera-mono-service/src/testFixtures/java/com/hedera/test/utils/KeyUtils.java

```diff
@@ -96,6 +97,18 @@ public static List<com.hederahashgraph.api.proto.java.Key> sanityRestored(List<?
                 .toList();
     }
 
+    public static List<com.hederahashgraph.api.proto.java.Key> sanityRestored(Set<? extends HederaKey> jKeys) {
```

Comment:
I've always hated this method name, ever since I first saw it. What does it mean anyway?  If it restores my sanity then it should be called restoreSanity. sanityRestored sounds like it should return a boolean (whether or not I am still insane), but then it would be clearer to call it something else `boolean isKimInsane() { return true; }`
This gripe is probably outside the scope of this PR.

Reply:
Agreed, I don't know what this method does either. To be addressed by another PR at some point